prov/efa/src/rxr/rxr_pkt_type.h

```diff
@@ -65,10 +65,11 @@
 #define RXR_READRSP_PKT		5
 #define RXR_RMA_CONTEXT_PKT	6
 #define RXR_EOR_PKT		7
+#define RXR_ATOMRSP_PKT         8
```

Comment:
Is this not part of v4 too, and aren't these PKT numbers append-only.

Reply:
Removed the confusing comments